src/helpers/timekeeperService.ts

```diff
@@ -17,8 +16,8 @@ export class TimekeeperService {
 
   private _currentPeriodStart: Date
   public getCurrentPeriod = (): Date => this._currentPeriodStart
-  public processBlock = (block: SubstrateBlock): boolean => {
-    const blockPeriodStart = getPeriodStart(block.timestamp)
+  public processBlock = (timestamp: Date): boolean => {
+    const blockPeriodStart = getPeriodStart(timestamp)
     const isNewPeriod = blockPeriodStart.valueOf() > this._currentPeriodStart.valueOf()
     if (isNewPeriod) this._currentPeriodStart = blockPeriodStart
```

Comment:
This should be separated for evm and substrate too.... Otherwise when running together l, only the fastest chain sets the pace for snapshots...

So we should keep the `_currentPeriodStart` information for each different chain to be snapshotted